### PR TITLE
Update XLA pin

### PR DESCRIPTION
**Pull request by IvanYashchuk**
Updating the pin to point to https://github.com/pytorch/xla/pull/3272.
It should resolve XLA failures introduced after merging https://github.com/pytorch/pytorch/pull/70988.

cc @bdhirsh

### CODE REVIEW

**Review by alanwaketan — APPROVED** [review]

File: .github/ci_commit_pins/xla.txt [hunk 1]
```diff
@@ -1 +1 @@
-frobenius_norm
+ac3973e96086a42232cd726626b8165cd13b6ee9
```
Comment (alanwaketan) [inline .github/ci_commit_pins/xla.txt:1]: You probably need the commit hash on the pt/xla master:
282567afcc41d2d9703d7002f6a435ec9fc51447

As pt/xla pin usually does't work with fork PR.

Comment (kit1980) [reply .github/ci_commit_pins/xla.txt]: I've updated.

**Review by kit1980 — APPROVED** [review]

**Review by JackCaoG — APPROVED** [review]

**Review by bdhirsh — APPROVED** [review]

**Comment by JackCaoG** [comment]
the pr doesn't make ci pass
```
/tmp/pytorch/xla/torch_xla/csrc/tensor_methods.cpp:2391:51: error: cannot define or redeclare 'linalg_eigh' here because namespace 'tensor_methods' does not enclose namespace 'XLATensor'
std::tuple<XLATensorPtr, XLATensorPtr> XLATensor::linalg_eigh(
                                       ~~~~~~~~~~~^
/tmp/pytorch/xla/torch_xla/csrc/tensor_methods.cpp:2394:28: error: use of undeclared identifier 'uplo_str'
  char uplo = std::toupper(uplo_str[0]);
                           ^
/tmp/pytorch/xla/torch_xla/csrc/tensor_methods.cpp:2396:66: error: no member named 'ops' in namespace 'torch::lazy'
  torch::lazy::NodePtr node = torch::lazy::MakeNode<torch::lazy::ops::SymEig>(
```

Can we make sure to check pytorch/xla build before merging the offending pr?

**Comment by kit1980** [comment]
> Can we make sure to check pytorch/xla build before merging the offending pr?

@JackCaoG so this one is not good https://github.com/pytorch/xla/pull/3272?
I saw it had build errors.

**Comment by JackCaoG** [comment]
> > Can we make sure to check pytorch/xla build before merging the offending pr?
> 
> @JackCaoG so this one is not good [pytorch/xla#3272](https://github.com/pytorch/xla/pull/3272)? I saw it had build errors.

no, the build still fail with this change, we need to work on another fix or revert the pytorch pr for now.

**Comment by kit1980** [comment]
> no, the build still fail with this change, we need to work on another fix or revert the pytorch pr for now.

I'll revert the PyTorch PR.

**Comment by JackCaoG** [comment]
@kit1980 Thanks, I will try to work on a fix in the mean time.

**Pull request closed without merging**